### DISCUSSION TITLE
This is about C++ and Python Data Science & Analytical Tools

### DISCUSSION THREAD

**Discussion by vsfeedback** [Microsoft/PTVS#7570] [Q&A] [answered]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/This-is-about-C-and-Python-Data-Scienc/1692740)._

---
If I want to do C++ programming.  Need to know which package I should install -

a)  Desktop Development with C++ (or)
b)  Universal Windows Platform Development

I need to know whether the Data Science & Analytical Tools package comes with Python Data Science libraries like NumPy, Pandas, Matplotlib, Scikit-learn, PyTorch, TensorFlow, Keras etc.

I also want to know how should I use form design with the Toolbox.

Regards,
Andrew Goh S M

---
#### Feedback Bot on 3/16/2022, 00:56 AM: 

(private comment, text removed)

**Comment by cwebster-99** [comment, marked as answer]
> If I want to do C++ programming. Need to know which package I should install

It depends on what you are trying to do. Could you provide more information? 

> I need to know whether the Data Science & Analytical Tools package comes with Python Data Science libraries like NumPy, Pandas, Matplotlib, Scikit-learn, PyTorch, TensorFlow, Keras etc.

No, it provides a clean Python install and you will need to install these packages on your own. We recommend using environments to manage packages with specific versions!
